src/code_translation/NodeSelector.py

```diff
@@ -131,6 +153,8 @@ def resolve_Call(self, node: cst.Call) -> Optional[Node]:
 
     def resolve_Name(self, node: cst.Name) -> Node:
         if node.value not in self.variables:
+            # TODO: Figure this out aswell
+            return node
```

Comment:
Why exactly was that necessary?

Reply:
![image](https://user-images.githubusercontent.com/70569959/214092122-80d0d8d9-b7f5-48f6-b2e4-275e2f0f9509.png)

It is still needed when we try to for example fill in the connection